tools/clang/lib/Headers/hlsl/README.txt

```diff
@@ -0,0 +1,7 @@
+HLSL Standard Header Library
+============================
+
+The contents of this directory and subdirectory are the HLSL Standard Header
```

Comment:
Is there only 1 subdirectory we care about?  Should this be 'The contents of this directory and _subdirectories_ are the HLSL Standard ....' ??